requirements/prod-requirements.txt

```diff
@@ -108,7 +108,7 @@ mock==2.0.0
 monotonic==1.5            # via eventlet
 msgpack-python==0.5.6
 ndg-httpsclient==0.5.1
-newrelic==4.6.0.106
+newrelic==4.8.0.110
```

Comment:
@snopoke or @dannyroberts  do we still use newrelic? It's in prod-requirements.in, but I thought that was removed long ago

Reply:
we were still using it for the apdex reports but since we've switched to datadog APM it's disabled. I think we can remove it.